.gitignore

```diff
@@ -1,4 +1,3 @@
-harbor
```

Comment:
does this ignore the directory contrib/helm/harbor ? I think harbor team ignores dir 'harbor' for some purpose. How about naming it 'contrib/helm-harbor'?

Reply:
in order to pass `helm lint` the directory the chart belongs in must be called `harbor`.  If the harbor team can provide the exact location of the `harbor` file they ignore I can make this ignore more specific.

Reply:
Got it. We can try adding '!/harbor' in contrib/helm/.gitignore or '!/contrib/helm/harbor' in .gitignore.